docs/excel/custom-functions-dialog.md

```diff
@@ -0,0 +1,105 @@
+---
+ms.date: 03/13/2019
+description: Create dialog boxes through custom functions in Excel using JavaScript.
+title: Custom functions dialogs (preview)
+localization_priority: Priority
+---
+# Display a dialog box in custom functions
+
+Within your add-in, you have two options for creating dialogs:
+
+- `OfficeRuntime.Dialog` can be accessed by both the browser-based JavaScript engine and custom functions runtime. You should use this dialog when developing custom functions.
+- The [Dialog API](https://docs.microsoft.com/office/dev/add-ins/develop/dialog-api-in-office-add-ins) is used by Office add-in's browser-based Javascript engine, but is not accessible from custom functions' runtime.
+
+The primary scenario for the `OfficeRuntime.Dialog` API is to facilitate authentication. For more details about authentication with custom functions, see [Custom functions authentication](./custom-functions-authentication.md).
+
+## Dialog API example
```

Comment:
This sample is unfortunately broken at the moment, but I'll submit a PR to fix it later from the PnP version I'm working on. Also will update this article to link to the PnP version as reference.